DropboxMoveFile/index.js

```diff
@@ -0,0 +1,21 @@
+const main = async (inputs, auths, event) => {
+  const fetch = require('isomorphic-fetch');
+  const { Dropbox } = require('dropbox');

```

Comment:
Move both require statements outside the main function and remove all semicolons.